CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.6)
+cmake_minimum_required(VERSION 3.8)
```

Comment:
`source_group` with support TREE & PREFIX is added in CMake 3.8

Reply:
we have https://github.com/nextcloud/desktop/pull/3714 waiting to be merged that is bumping to 3.16
I will merge it as soon as I am sure all our build platforms have the proper cmake version installed